xmtp_mls/src/groups/message_history.rs

```diff
@@ -384,58 +592,52 @@ pub(crate) fn decrypt_history_file(
     Ok(())
 }
 
-fn create_bundle_hash(
+async fn upload_history_bundle(
+    url: &str,
     file_path: PathBuf,
-    signing_key: &[u8],
-) -> Result<(Vec<u8>, String), MessageHistoryError> {
+) -> Result<String, MessageHistoryError> {
     let mut file = File::open(file_path)?;
     let mut content = Vec::new();
     file.read_to_end(&mut content)?;
 
-    let key = hmac::Key::new(hmac::HMAC_SHA256, signing_key);
-    let tag = hmac::sign(&key, &content);
-    let hmac_hex = hex::encode(tag.as_ref());
-    Ok((content, hmac_hex))
-}
-
-async fn upload_history_bundle(
-    url: &str,
-    file_path: PathBuf,
-    signing_key: &[u8],
-) -> Result<(), MessageHistoryError> {
-    let (content, hmac_hex) = create_bundle_hash(file_path, signing_key)?;
-
     let client = reqwest::Client::new();
-    let _response = client
-        .post(url)
-        .header("X-HMAC", hmac_hex)
-        .body(content)
-        .send()
-        .await?;
+    let response = client.post(url).body(content).send().await?;
 
-    Ok(())
+    if response.status().is_success() {
+        Ok(response.text().await?)
+    } else {
+        eprintln!(
```

Comment:
sorry last nit: prefer `tracing::error!()` here to be consistent with the rest of our logging

feel free to do that in one of the followups